cmd/cmdoptions/scenariorunner.go

```diff
@@ -0,0 +1,117 @@
+package cmdoptions
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/spf13/pflag"
+	"github.com/temporalio/omes/loadgen"
+	"go.temporal.io/sdk/client"
+	"go.uber.org/zap"
+)
+
+type ScenarioRunner struct {
```

Comment:
I do not think we should use CLI code outside of the CLI

Reply:
That's the purpose of this PR -- this PR changes the role of `ScenarioRunner` so that it can be used by library callers. So, when reviewing the PR, we can consider whether that's a good move or not, but it cannot be a premise of the review that it is not :)